packages/appstore/src/components/modals/wallet-dummy-success/wallet-dummy-success.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { useHistory } from 'react-router';
+import { Button, Modal, Text, DesktopWrapper, MobileDialog, MobileWrapper } from '@deriv/components';
+import { routes, formatMoney } from '@deriv/shared';
+import { localize, Localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import WalletAppCard from 'Components/wallet-app-card';
+import { getAccountName } from 'Constants/utils';
+import { useActiveWallet } from '@deriv/hooks';
+import './wallet-dummy-success.scss';
+
+const WalletDummySuccess = observer(() => {
+    const { ui, traders_hub } = useStore();
+    const { is_wallet_create_new_account_modal, setWalletCreateNewAccountModal } = traders_hub;
+
+    const { is_mobile } = ui;
+    const active_wallet = useActiveWallet();
+
+    const account_title = `${getAccountName({
+        account_type: 'trading',
+    })} (${active_wallet?.landing_company_name?.toUpperCase()})`;
+    const account_balance = formatMoney(active_wallet?.currency_config?.display_code, active_wallet?.balance, true);
```

Comment:
You can use display_balance from useAccountsList. Already formatted for ya 😸

Reply:
Mantap! Thanks man @thisyahlen-deriv for the suggestions 🙏🏼  didn't know this exist 😂